composer.json

```diff
@@ -10,5 +10,10 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zendframework": "~2.5"
+    },
+    "autoload": {
+        "psr-4": {
+          "Application\\": "module/Application/src/Application"
```

Comment:
Sure.. I replace with psr-0.. :) 

@adamlundrigan don't worry :) You are here to help me and I'm very happy to maintain this record! :trophy: